conanfile.py

```diff
@@ -0,0 +1,104 @@
+from conans import ConanFile, CMake, tools
+import os
+
```

Comment:
Is it a hard requirement for this file to be made available in the root directory of the repository ? 

If not, I am contemplating if it would be a good idea to move this to a folder `package_configs` at the root level of repository. This location may be potentially used any configuration files associated for other package managers too.

@umar456 @syurkevi What do you guys think ?

Reply:
(It's not a hard requirement at all, no)

Reply:
Lets leave it here for now and we can move it to another spot once we add additional files.